### PR TITLE
fix example2 crashes when camera did not provides exposure time

### PR DESCRIPTION
**Pull request by bugparty**

### CODE REVIEW

File: examples/eg2_properties.cpp [hunk 1]
```diff
@@ -22,20 +22,29 @@ void eg2_properties()
...
     camera.open(cameraDeivceList[0], 
         resolutions[resolutions.size() - 1].first,
         resolutions[resolutions.size() - 1].second
     );
-
+    std::cout << "show DirectShow properties" << std::string(*camera.getDirectShowProperties()) << std::endl;
     // Get exposure in second
-    std::cout << "Exposure: " + std::to_string(camera.getExposure()) + "s" << std::endl;
+    double exposureTime = camera.getExposure();
+    if (exposureTime > 0.0)
```
Comment (kcwongjoe) [inline examples/eg2_properties.cpp]: There has a function called supportExposure().

File: examples/eg2_properties.cpp [hunk 2]
```diff
@@ -22,20 +22,29 @@ void eg2_properties()
...
 
+
     // Open the first camera in the biggest resolution
     std::cout << "Open the first camera..." << std::endl;
     camera.open(cameraDeivceList[0], 
         resolutions[resolutions.size() - 1].first,
         resolutions[resolutions.size() - 1].second
     );
-
+    std::cout << "show DirectShow properties" << std::string(*camera.getDirectShowProperties()) << std::endl;
```
Comment (kcwongjoe) [inline examples/eg2_properties.cpp]: I want to keep the console clear. It could be great if you can remove it.

**Review by kcwongjoe — COMMENTED** [review]
Thanks for sharing. Can you make some change on that? Thanks

File: examples/eg2_properties.cpp (hunk 1 above)
Comment (bugparty) [reply examples/eg2_properties.cpp]: ok

File: examples/eg2_properties.cpp (hunk 2 above)
Comment (bugparty) [reply examples/eg2_properties.cpp]: ok,got it

**Pull request merged**